lsdcGui.py

```diff
@@ -4391,9 +4391,8 @@ def addRequestsToAllSelectedCB(self):
           self.selectedSampleID = itemData
           if (getBlConfig("queueCollect") == 0):
             if (self.mountedPin_pv.get() != self.selectedSampleID):                    
+              self.selectedSampleID = self.mountedPin_pv.get()
```

Comment:
is this meant to always queue a collection (because the return is removed), but always leave the sample selected?

Reply:
Correct, they felt it was unnecessarily stopping operations every time it happened, so the return was removed.

Reply:
ok, looks strange, but the beamlines asked for it so I'm ok with it.